.github/CODEOWNERS

```diff
@@ -23,3 +23,4 @@
 /apps/meteor/ee/client/omnichannel @RocketChat/omnichannel
 /apps/meteor/client/components/omnichannel @RocketChat/omnichannel
 /apps/meteor/client/components/voip @RocketChat/omnichannel
+/apps/meteor/app/statistics @RocketChat/sre @RocketChat/Architecture @RocketChat/Data
```

Comment:
github is complaining that both teams doesn't exist or don't have write access to the repo.. can you please make sure that's ok before we move forward? thx